.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,7 +19,7 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
 ## Proof that changes are correct
 
 <!--
-Add videos/screenshots of the user-facing interface in various display sizes (mainly phone, tablet, and desktop display size) to demonstrate that the changes made in this PR work correctly.
+Add videos/screenshots of the user-facing interface in various display sizes (mainly phone, tablet, and desktop display size) to demonstrate that the changes made in this PR work correctly. If you include a video make sure to also include the developer tools browser console, so that we can be sure that there are no console errors.
```

Comment:
Should we ask for console logs in screenshots as well? (i.e, all forms of proof should include logs?)